governance.md

```diff
@@ -0,0 +1,175 @@
+Chair or another Steering Committee member to do so.
+
+#### Vice-Chair
+
+In the event of the absence of the Chair, the Vice-Chair will preside at meetings. The Vice-Chair
+will be responsible for ensuring that the bi-monthly newsletter will be sent out.  In addition, the
+Vice-Chair will prepare an annual report of Association activities.
+
+#### Election Chair
+
+The Election Chair will be responsible for collecting nominations for Steering Committee elections,
+running such elections, and running other votes of the membership as required. The Election Chair
+will be filled by a Steering Committee member in the first year of their term for a duration of 1
+year.
+
+#### Secretary
+
+The Secretary will be responsible for ensuring that notes are taken in each meeting of the Committee
+and published after the meeting. In the absence of the Secretary, the presiding officer of the
+meeting will appoint another member to take on this responsibility.
+
+### Resignations
+
+A Steering Committee member may choose to resign by informing the Steering Committee Chair via
+email.  Additionally, the Steering Committee Chair can resign by informing the full committee via
+email. The resignation will take effect at the time specified in the notice, and, unless otherwise
+specified, formal acceptance of the resignation is not necessary.
+
+### Removal
+
+Steering Committee members can be removed by a two-thirds (⅔) vote of the full Steering Committee.
+In the event that that Committee is considering the removal of the Chair, the Vice-Chair will
+preside.
+
+### Vacancies
+
+If a seat on the Steering Committee becomes vacant, the remaining members of the Steering Committee
+may choose to appoint someone to fill the position for the remainder of the term. If the Steering
+Committee chooses to fill the position, the replacement will be chosen by a majority vote of the
+remaining members of the Steering Committee.  In the event that a majority vote cannot be achieved,
+the seat shall remain open until the end of the seat's term.
+
+## Committees, Working Groups, and Community Involvement
+
+Organizational functions of the US-RSE Association will be managed by standing committees that may
+consist of one or more members of the Steering Committee and interested members and participants in
+the Association. These organizational activities include communications (website, social media,
+etc.), conference planning, and others to be determined by the Steering Committee.
+
+Other activities, events, and efforts that have a fixed end date will be managed by working groups.
+These working groups can and should be initiated by members at large through discussion with the
+Steering Committee to ensure alignment with US-RSE priorities.  Working groups should have at least
+three (3) US-RSE members as participants.
+
+## Changes to Governance
+
+Minor revisions to this document can be made with the approval of at least two thirds of the current
```

Comment:
```suggestion
Minor revisions to this document can be made with the approval of at least two thirds (⅔) of the current
```

Reply:
Will there be any technical issue with using a non-ASCII character here?

Reply:
I wouldn't think so these days.